integrations/azure-devops/CHANGELOG.md

```diff
@@ -7,12 +7,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+# Port_Ocean 0.1.12 (2024-04-15)
+
+### Improvements
+
+- Added the default team of a project (PORT-7573)
+
+
```

Comment:
change, you add project kind as well as relation between repo and project, to get the team mirror property